src/SAWScript/Crucible/LLVM/X86.hs

```diff
@@ -566,6 +567,11 @@ assumePointsTo env tyenv nameEnv (LLVMPointsTo _ cond tptr tval) = do
   mem' <- liftIO $ C.LLVM.storeConstRaw sym mem ptr storTy C.LLVM.noAlignment val
   x86Mem .= mem'
 
+checkConcreteSizePointsToValue :: LLVMPointsToValue LLVMArch -> X86Sim (MS.SetupValue LLVM)
+checkConcreteSizePointsToValue = \case
+  ConcreteSizeValue val -> return val
+  SymbolicSizeValue{} -> throwX86 "unsupported x86_64 command: crucible_points_to_array_prefix"
```

Comment:
This error message is slightly misleading, because the command _does_ work as long as you give it a concrete size. Maybe something like "crucible_points_to_array_prefix is unsupported for symbolic sizes when verifying x86 code"?

Reply:
technically, `crucible_points_to_array_prefix` is represented internally as `SymbolicSizeValue`, regardless of whether the size is concrete.

Reply:
Ah, okay.